c++/src/kj/async.c++

```diff
@@ -1047,15 +1056,20 @@ FiberStack::FiberStack(size_t stackSizeParam)
 #elif !__BIONIC__
   // Note: Nothing below here can throw. If that changes then we need to call Impl::free(impl)
   //   on exceptions...
+  ucontext_t context;
+  impl = Impl::alloc(stackSize, &context);
 
   // POSIX says the arguments are ints, not pointers. So we split our pointer in half in order to
   // work correctly on 64-bit machines. Gross.
   uintptr_t ptr = reinterpret_cast<uintptr_t>(this);
   int arg1 = ptr & ((uintptr_t(1) << (sizeof(ptr) * 4)) - 1);
   int arg2 = ptr >> (sizeof(ptr) * 4);
 
-  makecontext(&impl.fiberContext, reinterpret_cast<void(*)()>(&StartRoutine::run), 2, arg1, arg2);
+  makecontext(&context, reinterpret_cast<void(*)()>(&StartRoutine::run), 2, arg1, arg2);
 
+  if (setjmp(impl->originalJmpBuf) == 0) {
```

Comment:
Oops, should be `_setjmp`?

Reply:
Sorry for the run-around. Feel free to use `--amend` for this one fix, then should be good to merge.

Reply:
Hmm how did that get there.

Reply:
Fixed.